### ISSUE TITLE
Design mentor gamification system

### ISSUE THREAD

**Issue by hollomancer** [OperationCode/mentorship#37]

**Comment by jdspringr07** [comment]
@hollomancer Could you please elaborate on this? Is it some kind of rewards based system for mentors that excel in their role?

**Comment by hollomancer** [comment]
Yeah. This is one of those things that I wanted to defer until "once we got big enough..."

That time may have come.

**Comment by rickr** [comment]
This would be a way to increase engagement for mentors, probably trying to incorporate a little competition. 

https://en.m.wikipedia.org/wiki/Gamification

We'll want to be careful... This is a trend so we'll want to make sure we do it well.
